src/test/java/org/humancellatlas/ingest/schemas/SchemaTest.java

```diff
@@ -0,0 +1,77 @@
+package org.humancellatlas.ingest.schemas;
+
+import com.github.tomakehurst.wiremock.junit.WireMockRule;
+import org.humancellatlas.ingest.schemas.schemascraper.SchemaScraper;
+import org.humancellatlas.ingest.schemas.schemascraper.impl.SchemaScraperImpl;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import java.io.File;
+import java.net.URI;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.Arrays;
+import java.util.Collection;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.*;
+
+
+/**
+ * Created by rolando on 19/04/2018.
+ */
+@RunWith(SpringRunner.class)
+@SpringBootTest
+public class SchemaTest {
+    @Autowired SchemaService schemaService;
+
+    @Rule
+    public WireMockRule wireMockRule = new WireMockRule(8088);
+
+    @Test
+    public void testSchemaScrape() throws Exception {
+        // given
+        // an s3 bucket files listing as XML
+        SchemaScraper schemaScraper = new SchemaScraperImpl();
+
+        // when
+        stubFor(
+                get(urlEqualTo("/"))
+                        .willReturn(aResponse()
+                                            .withStatus(200)
+                                            .withHeader("Content-Type", "application/xml")
+                                            .withBody(new String(Files.readAllBytes(Paths.get(new File(".").getAbsolutePath() + "/src/test/java/org/humancellatlas/ingest/schemas/testfiles/TestBucketListing.xml"))))));
+
```

Comment:
Test files should be in `src/test/resources`. Only `*.java` files (or anything compiled into bytecodes) should be in `src/test/java`.